Attaching to stockerservice_redis_1, stockerservice_celery_1, stockerservice_website_1
[36mredis_1    |[0m 1:C 21 Nov 17:05:11.673 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
[36mredis_1    |[0m 1:C 21 Nov 17:05:11.674 # Redis version=4.0.11, bits=64, commit=00000000, modified=0, pid=1, just started
[36mredis_1    |[0m 1:C 21 Nov 17:05:11.674 # Configuration loaded
[36mredis_1    |[0m 1:M 21 Nov 17:05:11.685 * Running mode=standalone, port=6379.
[36mredis_1    |[0m 1:M 21 Nov 17:05:11.685 # WARNING: The TCP backlog setting of 511 cannot be enforced because /proc/sys/net/core/somaxconn is set to the lower value of 128.
[36mredis_1    |[0m 1:M 21 Nov 17:05:11.685 # Server initialized
[36mredis_1    |[0m 1:M 21 Nov 17:05:11.685 # WARNING you have Transparent Huge Pages (THP) support enabled in your kernel. This will create latency and memory usage issues with Redis. To fix this issue run the command 'echo never > /sys/kernel/mm/transparent_hugepage/enabled' as root, and add it to your /etc/rc.local in order to retain the setting after a reboot. Redis must be restarted after THP is disabled.
[36mredis_1    |[0m 1:M 21 Nov 17:05:11.685 * Ready to accept connections
[36mredis_1    |[0m 1:C 22 Nov 04:32:07.157 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
[36mredis_1    |[0m 1:C 22 Nov 04:32:07.160 # Redis version=4.0.11, bits=64, commit=00000000, modified=0, pid=1, just started
[36mredis_1    |[0m 1:C 22 Nov 04:32:07.160 # Configuration loaded
[36mredis_1    |[0m 1:M 22 Nov 04:32:07.164 * Running mode=standalone, port=6379.
[36mredis_1    |[0m 1:M 22 Nov 04:32:07.164 # WARNING: The TCP backlog setting of 511 cannot be enforced because /proc/sys/net/core/somaxconn is set to the lower value of 128.
[36mredis_1    |[0m 1:M 22 Nov 04:32:07.164 # Server initialized
[36mredis_1    |[0m 1:M 22 Nov 04:32:07.164 # WARNING you have Transparent Huge Pages (THP) support enabled in your kernel. This will create latency and memory usage issues with Redis. To fix this issue run the command 'echo never > /sys/kernel/mm/transparent_hugepage/enabled' as root, and add it to your /etc/rc.local in order to retain the setting after a reboot. Redis must be restarted after THP is disabled.
[36mredis_1    |[0m 1:M 22 Nov 04:32:07.164 * Ready to accept connections
[36mredis_1    |[0m 1:signal-handler (1542861178) Received SIGTERM scheduling shutdown...
[36mredis_1    |[0m 1:M 22 Nov 04:32:58.363 # User requested shutdown...
[36mredis_1    |[0m 1:M 22 Nov 04:32:58.363 # Redis is now ready to exit, bye bye...
[36mredis_1    |[0m 1:C 22 Nov 05:30:47.626 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
[36mredis_1    |[0m 1:C 22 Nov 05:30:47.627 # Redis version=4.0.11, bits=64, commit=00000000, modified=0, pid=1, just started
[36mredis_1    |[0m 1:C 22 Nov 05:30:47.627 # Configuration loaded
[36mredis_1    |[0m 1:M 22 Nov 05:30:47.634 * Running mode=standalone, port=6379.
[36mredis_1    |[0m 1:M 22 Nov 05:30:47.635 # WARNING: The TCP backlog setting of 511 cannot be enforced because /proc/sys/net/core/somaxconn is set to the lower value of 128.
[36mredis_1    |[0m 1:M 22 Nov 05:30:47.635 # Server initialized
[36mredis_1    |[0m 1:M 22 Nov 05:30:47.635 # WARNING you have Transparent Huge Pages (THP) support enabled in your kernel. This will create latency and memory usage issues with Redis. To fix this issue run the command 'echo never > /sys/kernel/mm/transparent_hugepage/enabled' as root, and add it to your /etc/rc.local in order to retain the setting after a reboot. Redis must be restarted after THP is disabled.
[36mredis_1    |[0m 1:M 22 Nov 05:30:47.635 * Ready to accept connections
[36mredis_1    |[0m 1:signal-handler (1542864692) Received SIGTERM scheduling shutdown...
[36mredis_1    |[0m 1:M 22 Nov 05:31:32.306 # User requested shutdown...
[36mredis_1    |[0m 1:M 22 Nov 05:31:32.306 # Redis is now ready to exit, bye bye...
[36mredis_1    |[0m 1:C 22 Nov 05:31:38.367 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
[36mredis_1    |[0m 1:C 22 Nov 05:31:38.368 # Redis version=4.0.11, bits=64, commit=00000000, modified=0, pid=1, just started
[36mredis_1    |[0m 1:C 22 Nov 05:31:38.369 # Configuration loaded
[36mredis_1    |[0m 1:M 22 Nov 05:31:38.374 * Running mode=standalone, port=6379.
[36mredis_1    |[0m 1:M 22 Nov 05:31:38.374 # WARNING: The TCP backlog setting of 511 cannot be enforced because /proc/sys/net/core/somaxconn is set to the lower value of 128.
[36mredis_1    |[0m 1:M 22 Nov 05:31:38.375 # Server initialized
[36mredis_1    |[0m 1:M 22 Nov 05:31:38.375 # WARNING you have Transparent Huge Pages (THP) support enabled in your kernel. This will create latency and memory usage issues with Redis. To fix this issue run the command 'echo never > /sys/kernel/mm/transparent_hugepage/enabled' as root, and add it to your /etc/rc.local in order to retain the setting after a reboot. Redis must be restarted after THP is disabled.
[36mredis_1    |[0m 1:M 22 Nov 05:31:38.375 * Ready to accept connections
[32mwebsite_1  |[0m [2018-11-22 05:31:57 +0000] [10] [INFO] Starting gunicorn 19.9.0
[32mwebsite_1  |[0m [2018-11-22 05:31:57 +0000] [10] [INFO] Listening at: http://127.0.0.1:8000 (10)
[32mwebsite_1  |[0m [2018-11-22 05:31:57 +0000] [10] [INFO] Using worker: sync
[32mwebsite_1  |[0m [2018-11-22 05:31:57 +0000] [13] [INFO] Booting worker with pid: 13
[32mwebsite_1  |[0m [2018-11-22 05:31:57 +0000] [13] [ERROR] Exception in worker process
[32mwebsite_1  |[0m Traceback (most recent call last):
[32mwebsite_1  |[0m   File "/opt/conda/lib/python3.6/site-packages/gunicorn/arbiter.py", line 583, in spawn_worker
[32mwebsite_1  |[0m     worker.init_process()
[32mwebsite_1  |[0m   File "/opt/conda/lib/python3.6/site-packages/gunicorn/workers/base.py", line 129, in init_process
[32mwebsite_1  |[0m     self.load_wsgi()
[32mwebsite_1  |[0m   File "/opt/conda/lib/python3.6/site-packages/gunicorn/workers/base.py", line 138, in load_wsgi
[32mwebsite_1  |[0m     self.wsgi = self.app.wsgi()
[32mwebsite_1  |[0m   File "/opt/conda/lib/python3.6/site-packages/gunicorn/app/base.py", line 67, in wsgi
[32mwebsite_1  |[0m     self.callable = self.load()
[32mwebsite_1  |[0m   File "/opt/conda/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 52, in load
[32mwebsite_1  |[0m     return self.load_wsgiapp()
[32mwebsite_1  |[0m   File "/opt/conda/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 41, in load_wsgiapp
[32mwebsite_1  |[0m     return util.import_app(self.app_uri)
[32mwebsite_1  |[0m   File "/opt/conda/lib/python3.6/site-packages/gunicorn/util.py", line 350, in import_app
[32mwebsite_1  |[0m     __import__(module)
[32mwebsite_1  |[0m ModuleNotFoundError: No module named 'application'
[32mwebsite_1  |[0m [2018-11-22 05:31:57 +0000] [13] [INFO] Worker exiting (pid: 13)
[32mwebsite_1  |[0m [2018-11-22 05:31:57 +0000] [10] [INFO] Shutting down: Master
[32mwebsite_1  |[0m [2018-11-22 05:31:57 +0000] [10] [INFO] Reason: Worker failed to boot.
[33mcelery_1   |[0m Traceback (most recent call last):
[33mcelery_1   |[0m   File "/opt/conda/bin/celery", line 11, in <module>
[33mcelery_1   |[0m     sys.exit(main())
[33mcelery_1   |[0m   File "/opt/conda/lib/python3.6/site-packages/celery/__main__.py", line 16, in main
[33mcelery_1   |[0m     _main()
[33mcelery_1   |[0m   File "/opt/conda/lib/python3.6/site-packages/celery/bin/celery.py", line 322, in main
[33mcelery_1   |[0m     cmd.execute_from_commandline(argv)
[33mcelery_1   |[0m   File "/opt/conda/lib/python3.6/site-packages/celery/bin/celery.py", line 496, in execute_from_commandline
[33mcelery_1   |[0m     super(CeleryCommand, self).execute_from_commandline(argv)))
[33mcelery_1   |[0m   File "/opt/conda/lib/python3.6/site-packages/celery/bin/base.py", line 273, in execute_from_commandline
[33mcelery_1   |[0m     argv = self.setup_app_from_commandline(argv)
[33mcelery_1   |[0m   File "/opt/conda/lib/python3.6/site-packages/celery/bin/base.py", line 479, in setup_app_from_commandline
[33mcelery_1   |[0m     self.app = self.find_app(app)
[33mcelery_1   |[0m   File "/opt/conda/lib/python3.6/site-packages/celery/bin/base.py", line 501, in find_app
[33mcelery_1   |[0m     return find_app(app, symbol_by_name=self.symbol_by_name)
[33mcelery_1   |[0m   File "/opt/conda/lib/python3.6/site-packages/celery/app/utils.py", line 359, in find_app
[33mcelery_1   |[0m     sym = symbol_by_name(app, imp=imp)
[33mcelery_1   |[0m   File "/opt/conda/lib/python3.6/site-packages/celery/bin/base.py", line 504, in symbol_by_name
[33mcelery_1   |[0m     return imports.symbol_by_name(name, imp=imp)
[33mcelery_1   |[0m   File "/opt/conda/lib/python3.6/site-packages/kombu/utils/imports.py", line 56, in symbol_by_name
[33mcelery_1   |[0m     module = imp(module_name, package=package, **kwargs)
[33mcelery_1   |[0m   File "/opt/conda/lib/python3.6/site-packages/celery/utils/imports.py", line 104, in import_from_cwd
[33mcelery_1   |[0m     return imp(module, package=package)
[33mcelery_1   |[0m   File "/opt/conda/lib/python3.6/importlib/__init__.py", line 126, in import_module
[33mcelery_1   |[0m     return _bootstrap._gcd_import(name[level:], package, level)
[33mcelery_1   |[0m   File "<frozen importlib._bootstrap>", line 994, in _gcd_import
[33mcelery_1   |[0m   File "<frozen importlib._bootstrap>", line 971, in _find_and_load
[33mcelery_1   |[0m   File "<frozen importlib._bootstrap>", line 941, in _find_and_load_unlocked
[33mcelery_1   |[0m   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
[33mcelery_1   |[0m   File "<frozen importlib._bootstrap>", line 994, in _gcd_import
[33mcelery_1   |[0m   File "<frozen importlib._bootstrap>", line 971, in _find_and_load
[33mcelery_1   |[0m   File "<frozen importlib._bootstrap>", line 953, in _find_and_load_unlocked
[33mcelery_1   |[0m ModuleNotFoundError: No module named 'stocker_server'
[32mstockerservice_website_1 exited with code 3
[0m[33mstockerservice_celery_1 exited with code 1
[0mGracefully stopping... (press Ctrl+C again to force)
